runtime/stringbuf.c

```diff
@@ -383,40 +382,15 @@ rsRetVal rsCStrSetSzStr(cstr_t *pThis, uchar *pszNew)
  */
 uchar*  rsCStrGetSzStrNoNULL(cstr_t *pThis)
 {
-	size_t i;
-
 	rsCHECKVALIDOBJECT(pThis, OIDrsCStr);
 
 	if(pThis->pBuf == NULL)
 		return (uchar*) "";
 
 	if(pThis->pBuf != NULL)
```

Comment:
thx, indeed this is not necessary :-) Will ammend the commit.

side-note: if I have time the next two days, I'll see that I do some more refactoring, there is sufficient potential ;)